slices/perl-modules-5.38.yaml

```diff
@@ -0,0 +1,16 @@
+package: perl-modules-5.38
+
+essential:
+  - perl-modules-5.38_copyright
+
+slices:
+  modules:
+    essential:
+      - perl-base_libs
+      - perl-base_modules
+    contents:
+      /usr/share/perl/5.38*/**:
```

Comment:
This glob is too broad. I am wondering whether we should list each perl package/module per line, something like below:

```
/usr/share/perl/5.38*/AnyDBM_File.pm
/usr/share/perl/5.38*/App/**
```

Maybe we should do the same for other packages before where we are using broad globs like this.

Reply:
I think there are many more things then `/usr/share/perl/5.38*/App/**`. We have, e.g. `B/`, `Benchmark.pm`, `CORE.pod`, `CPAN/`, ... in `/usr/share/perl/5.38*`. It would be too much if we explicit them without using such glob.